src/main/scala/org/apache/spark/deploy/armada/Config.scala

```diff
@@ -405,6 +380,56 @@ private[spark] object Config {
       .stringConf
       .createOptional
 
+  val ARMADA_DELETE_EXECUTORS: ConfigEntry[Boolean] =
```

Comment:
not used in this PR

Reply:
https://github.com/armadaproject/armada-spark/blob/525da4c4536fccc64338636efb863cbdf8c89cf9/src/main/scala/org/apache/spark/scheduler/cluster/armada/ArmadaClusterManagerBackend.scala#L212